life/modernized/24-chapter-24.adoc

```diff
@@ -655,6 +724,9 @@ and deputed William Penn, in conjunction with some others, to present it.
 Being admitted to the king,
 William Penn introduced the address by a speech to the following import.
 
+[.embedded-content-document]
```

Comment:
I wasn't sure if this should be a quote block since it isn't a document?? I know these always give us trouble. Same thing for the king's reply on line 798.